bsip-0071.md

```diff
@@ -51,59 +75,126 @@ In my view, a good way to handle bad debt need to follow below principles:
 
 In above mentioned 5 solutions, only "5. Prevent Global Settlement" fulfills all 3 principles.
 
-Partial GS is another attractive solution: it is similar to Global Settlement but differs in that (a) only takes over the bad debt positions and moves them to a settlement pool without touching the debt positions with CR>1, and (b) users can issue force settlement from the pool, from the margin call orders, or from the good debt position depending on which has the lowest CR.
+Partial GS is another attractive solution: it is similar to Global Settlement but differs in that 
+(a) only takes over the bad debt positions and moves them to a settlement pool without touching the debt 
+positions with CR>1, and 
+(b) users can issue force settlement from the pool, from the margin call orders, or from the good debt position 
+depending on which has the lowest CR.
 
 The problem of Partial GS is that it does not resolve itself automatically when the price recovers.
 
-The "Prevent Global Settlement" solution adopts a new idea to handle bad debt: while bad debt appears, the smartcoin will be devaluated at a ratio of the lowest CR from among the bad debt positions, yet all the smartcoin trading features, including borrowing, margin call and force settlement, will all switch to refer to GS price to ensure the continuity and fairness of all the features.
+The "Prevent Global Settlement" solution adopts a new idea to handle bad debt: 
+while bad debt appears, the smartcoin will be devaluated at a ratio of the lowest CR from among the bad debt positions, 
+yet all the smartcoin trading features, including borrowing, margin call and force settlement, 
+will all switch to refer to GS price to ensure the continuity and fairness of all the features.
 
 # Specifications
 
-Add one flag "Prevent Global Settlement" to each smartcoin asset.
+#### Add flags "Prevent Global Settlement" and "Set Threshhold price" to each smartcoin asset.
+
+Those flags are set by the asset owner, and come with their respective permission like all others. 
+Default values 
```

Comment:
The "Default values" thought is incomplete

Reply:
Done